langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -197,6 +275,11 @@ public isolated function data(xml x) returns string = @java:Method {
 
 # Returns the target part of the processing instruction.
 #
+# ```ballerina
+# xml:ProcessingInstruction p = xml `<?website wso2.com?>`;
+# p.getTarget() ⇒ website
```

Comment:
```suggestion
# xml:ProcessingInstruction p = xml `<?sort ascending?>`;
# p.getTarget() ⇒ sort
```

Reply:
Updated in Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0